master/buildbot/status/master.py

```diff
@@ -129,6 +129,12 @@ def getTitleURL(self):
     def getBuildbotURL(self):
         return self.master.config.buildbotURL
 
+    def getBuildbotURLAliases(self):
```

Comment:
Please add unit tests for this function.

list.append() does not return the list